### ISSUE TITLE
NuGet package doesn't contain MSEngine.Solver

### ISSUE THREAD

**Issue by ndossche** [bradmarder/MSEngine#1]
Hi, thanks for your work on this library.

It seems that the NuGet package only contains the MSEngine.Core namespace, and not the MSEngine.Solver, or am I missing something?

**Comment by bradmarder** [comment]
@nielsdos You are correct, the solver isn't included in the NuGet package. I will try to find some time today to investigate adding it in by default.

**Comment by ndossche** [comment]
Thanks!

**Comment by bradmarder** [comment]
@nielsdos Unfortunately, I didn't find a quick solution, and I don't have time right now to investigate. I realize it's not ideal, but you could just copy/paste the .csproj/files from the MSEngine.Solver project as a temporary solution.

If there is any way I can help you with integrating this library, please let me know and I'll be more than glad to help. I understand the API can be difficult to use (I've sacrificed a lot of readability for perf). The examples in in benchmarks/bot/console.app are a decent start.

**Comment by ndossche** [comment]
No worries! I simply copied the code as you mentioned and it worked great.
I was able to get it working quickly with the sample code of benchmarks and the console app :).
